source/speakers/_index.md

```diff
@@ -50,3 +64,4 @@ at Apache), please email us and suggest we include your slides.
   [6]: https://www.apache.org/foundation/press/kit/
   [7]: slides.html
   [8]: themes
+  [9]: https://lists.apache.org/list?community@apache.org
```

Comment:
Thank you @rbowen for adding this section. A quick one, should we mention `https://lists.apache.org/list.html?dev@community.apache.org` list instead of  `https://lists.apache.org/list?community@apache.org`. 
I am assuming it's a typo but just want to make sure.

Reply:
Thanks. Fixed.